### DISCUSSION TITLE
Blender crashes

### DISCUSSION THREAD

**Discussion by TheRealSpacerex** [GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon#392] [General]
I was thrilled to see this because I've been lumbering along with my CPU only rendering in Blender. I followed the install instructions at https://youtu.be/Mm0Vtoy8hhY but when I click the box in addons, (the final step) blender crashes. I have tried it with 2.90.1 and 3.1.0 with same result.

 Model Name:	Mac Pro
  Model Identifier:	MacPro5,1
  Processor Name:	6-Core Intel Xeon
  Processor Speed:	3.33 GHz
  Number of Processors:	1
  Total Number of Cores:	6
  L2 Cache (per Core):	256 KB
  L3 Cache:	12 MB
  Memory:	48 GB
  Boot ROM Version:	MP51.0084.B00
  SMC Version (system):	1.39f11
  SMC Version (processor tray):	1.39f11
  Serial Number (system):	C07JK043F4MG
  Serial Number (processor tray):	J524100D5CZJC     
  Hardware UUID:	AF28E1BE-F54B-5EAF-845D-C91F8E10A0A0

Chipset Model:	Radeon RX 580
  Type:	GPU
  Bus:	PCIe
  Slot:	Slot-1
  PCIe Lane Width:	x16
  VRAM (Dynamic, Max):	8192 MB
  Vendor:	AMD (0x1002)
  Device ID:	0x67df
  Revision ID:	0x00e7
  Metal:	Supported, feature set macOS GPUFamily1 v3
  Displays:
Cinema:
  Display Type:	LCD
  Resolution:	1050 x 1680
  UI Looks like:	1050 x 1680
  Framebuffer Depth:	24-Bit Color (ARGB8888)
  Display Serial Number:	2A5420A6UFZ 
  Mirror:	Off
  Online:	Yes
  Rotation:	270
  Automatically Adjust Brightness:	No
LED Cinema Display:
  Display Type:	LCD
  Resolution:	2560 x 1440 (QHD/WQHD - Wide Quad High Definition)
  UI Looks like:	2560 x 1440
  Framebuffer Depth:	24-Bit Color (ARGB8888)
  Display Serial Number:	W823620X6JL 
  Main Display:	Yes
  Mirror:	Off
  Online:	Yes
  Rotation:	Supported
  Automatically Adjust Brightness:	No
  Connection Type:	DisplayPort

**Comment by bsavery** [comment]
What version of macOS?

**Reply by TheRealSpacerex** [reply]
10.13.6

**Reply by bsavery** [reply]
We only support Mojave and beyond
